core/client/src/main/java/alluxio/client/block/RetryHandlingBlockWorkerClient.java

```diff
@@ -63,7 +65,9 @@
       ThreadFactoryUtils.build("block-worker-heartbeat-%d", true));
   private static final ExecutorService HEARTBEAT_CANCEL_POOL = Executors.newFixedThreadPool(5,
       ThreadFactoryUtils.build("block-worker-heartbeat-cancel-%d", true));
-
+  private static final WriteTier CLIENT_WRITE_TIER_POLICY =
```

Comment:
My reasoning behind naming this a policy is because the client isn't just choosing a tier. The resulting tier selected is determined by the client's selection and the tiers that exist on a worker. For example, with LOWEST, that could result in writing tier 0, 1, or 2.

I'm happy to not use the word policy in this patch if it's making things more confusing.